lib/web_console/templates/console.js.erb

```diff
@@ -864,11 +864,23 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.outer.scrollTop = this.outer.scrollHeight;
 };
 
-// Change the binding of the console
+/**
+ * Change the binding of the console
+ * Also updates a div to show that it has been changed.
+ */
 REPLConsole.prototype.switchBindingTo = function(frameId, callback) {
   var url = this.getSessionUrl('trace');
   var params = "frame_id=" + encodeURIComponent(frameId);
-  postRequest(url, params, callback);
+  var notification = document.createElement('div');
+  var _this = this;
+
+  notification.className = "console-notification";
+  postRequest(url, params, function() {
+    notification.innerHTML = "Binding has changed to: " + callback();
```

Comment:
I'm thinking whether we call this `Context has changed to:` because we cannot expect the web console users to know about bindings or the terminology around how code evaluation works in Ruby.

Reply:
I initially had it as **Context** as well but renamed it last minute. I guess I can change it back! I agree it sounds more "intuitive" for people using web-console.